services/user-management/src/main/java/com/foodie/user/model/service/impl/RoleServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package com.foodie.user.model.service.impl;
+
+import java.util.UUID;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.foodie.FoodieBaseRepositoryImpl;
+import com.foodie.user.model.Role;
+import com.foodie.user.model.repositories.RoleRepository;
+import com.foodie.user.model.service.RoleService;
+
+@Service
+@Transactional
+public class RoleServiceImpl extends FoodieBaseRepositoryImpl<Role, UUID> implements RoleService {
```

Comment:
I'm not sure what the plan is here, but this smells bad

Reply:
https://stackoverflow.com/questions/19417670/using-generics-in-spring-data-jpa-repositories